vortex-array/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(once_cell_try)]
```

Comment:
in other places we use the once_cell crate to get this feature without requiring one more nightly flag.

Reply:
Yeah let's chat Monday, there might be some convergence. I also want to make FilterMask internally memoize the indices or slices, instead of necessarily a boolean buffer.